### ISSUE TITLE
Wave disappear after onStop is called

### ISSUE THREAD

**Issue by Muhammad-Adam** [cleandersonlobo/react-mic#10]
```
<ReactMic
    record={record}
    className={styles.recordBar}
    onStop={onStop}
    strokeColor={"#001529"}
    backgroundColor={"#FFFFFF"}
/>
```

my ```onStop``` function is just storing the data in some state.

When my ```record``` boolean changed to ```false```, the wave disappears.

**Comment by nemesis1346** [comment]
Yes I am having the same issue. After stop, the wave component disappears. Any solution?

**Comment by cropcrop** [comment]
Same problem here: https://codesandbox.io/s/ecstatic-dust-480ub

**Comment by danielsGit** [comment]
Same issue, here! Any solution?

**Comment by gracew** [comment]
Using version 1.1.2 of the package (prior to [this commit](https://github.com/cleandersonlobo/react-mic/commit/5cc084e0fe4353115355142a6f097e2746830971)) solves the problem of the wave disappearing. Not sure about the memory implications of using this version though.
